src/openWindow.java

```diff
@@ -20,7 +20,7 @@
 
     /** Creates new form openWindow */
     public openWindow() {
-        
+        int abc;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "abc" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1481)